groups/status.go

```diff
@@ -3,7 +3,12 @@
 
 package groups
 
-import svcerr "github.com/absmach/magistrala/pkg/errors/service"
+import (
+	"encoding/json"
+	"strings"
+
+	svcerr "github.com/absmach/magistrala/pkg/errors/service"
+)
 
 // Status represents Group status.
 type Status uint8
```

Comment:
We are repeating Status across services. Since all the entities have the same status options, please extract to the common package so we don't reimplement per-service.

Reply:
@dborovcanin I agree, Can we do it in different PR. 
Because it touches all the service ?

Reply:
We can do this in a separate PR.